.github/workflows/node.js.yml

```diff
@@ -23,6 +23,6 @@ jobs:
       - name: Deploy 🚀
         uses: JamesIves/github-pages-deploy-action@releases/v3
         with:
-          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }} # Use a different Personal Access token personal access token you generated with at least public_repo, read:org, workflow access by adding it to the Settings/Secrets part of this repository with anme ACCESS_TOKEN
```

Comment:
```suggestion
          ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This is provided by GitHub.
```

Reply:
I made this change suggestion, I do want to point to https://github.com/saadpasta/developerFolio/pull/228#discussion_r470995005 as reference.  Since we do include the link in the `README.md` pointing developers to the JamesIves project I am ok with this.  I do wish there was some more obvious specifications of the `GITHUB_TOKEN` vs the `ACCESS_TOKEN`.  I couldn't quite find the permission differences when it came to repository commits to the `gh-pages` branch during the action process.

Reply:
> 

We need to revert this change